examples/basic/README.md

```diff
@@ -0,0 +1,47 @@
+# Basic example
+
+
+# Outputs
+```
+Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+module_tags = {
+  "name" = "PROD-SBLACK4-TAGS"
+  "tags" = {
+    "Name" = "PROD-SBLACK4-TAGS"
+    "terraform_applied_at" = "2019-11-14T16:41:04Z"
+    "terraform_managed" = true
+    "terraform_module" = ".."
+    "terraform_root_module" = "."
+    "terraform_workspace" = "default"
+  }
+  "tags_as_list_of_maps" = [
+    {
+      "key" = "Name"
+      "value" = "PROD-SBLACK4-TAGS"
+    },
+    {
+      "key" = "terraform_applied_at"
+      "value" = "2019-11-14T16:41:04Z"
+    },
+    {
+      "key" = "terraform_managed"
+      "value" = true
+    },
+    {
+      "key" = "terraform_module"
+      "value" = ".."
+    },
+    {
+      "key" = "terraform_root_module"
+      "value" = "."
+    },
+    {
+      "key" = "terraform_workspace"
+      "value" = "default"
+    },
+  ]
+}
+```
```

Comment:
let's def try to keep the example updated, it'd be nice if we had this done automatically on merge or something